### PR TITLE
Set image alt attribute to empty string if media text is null or empty

### LINKED ISSUE #14382: Media text should be blank when field is blank
When a media field has "Allow media text" enabled, if the Media Text field is empty, the alt text is rendered as the file / path of the image. 

### Expected behavior
The alt tag should be blank when the Media Text field is empty. This is useful for cases where the image is for presentation purposes only and should not be read by a screen reader. Source regarding decorative images: https://www.w3.org/WAI/tutorials/images/decorative/

### PR DESCRIPTION
**Pull request by DrewBrasher**
fixes #14382

### CODE REVIEW

**Review by sebastienros — APPROVED** [review]

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml [hunk 1]
```diff
@@ -15,10 +15,11 @@
             var path = paths[i];
             var url = Orchard.AssetUrl(path);
             var text = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : path;
+            var alt = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : string.Empty;
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml:18]: Can we move the line 18 in the else block "line 25"?

**Review by MikeAlhayek — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml (hunk 1 above)
Comment (DrewBrasher) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml:18]: It would need to be in the if block on line 22.

Comment (DrewBrasher) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml]: @MikeAlhayek how does https://github.com/OrchardCMS/OrchardCore/pull/16859/commits/63928d7cb9a87e2025d4b081d5ac83feda2d1ea6  look?

Comment (DrewBrasher) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml]: I could move the `var text` assignment (line 17) to inside the else block now that it isn't used in the if block.

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml]: @DrewBrasher 
Yes also please move line 17 "var text" to the else block. 

Additionally, please change paths.Any() on line 9 to paths.length > 0

Comment (DrewBrasher) [reply src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml]: I made those changes

File: src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml [hunk 2]
```diff
@@ -6,22 +6,23 @@
...
         {
             var path = paths[i];
             var url = Orchard.AssetUrl(path);
-            var text = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : path;
 
             if (Orchard.IsImageFile(path))
             {
-                <img src="@url" alt="@text">
+                var alt = !string.IsNullOrEmpty(mediaTexts.ElementAtOrDefault(i)) ? mediaTexts[i] : string.Empty;
+                <img src="@url" alt="@alt">
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml:21]:
```suggestion
                <img src="@url" alt="@alt" />
```

**Review by MikeAlhayek — APPROVED** [review]

**Pull request merged**